data-serving/scripts/export-data/common.sh

```diff
@@ -0,0 +1,3 @@
+ECR=612888738066.dkr.ecr.us-east-1.amazonaws.com
+# ingestion role contains necessary permissions to access S3 buckets
+JOB_ROLE_ARN="arn:aws:iam::612888738066:role/gdh-ingestion-job-role"
```

Comment:
Is there a different approach we can use for configuration management? I would like changing the variables to be an explicit decision rather than something that happens when a PR gets merged or new software gets deployed, as different parts get deployed through different means.